vendor/github.com/coreos/etcd/raft/raftpb/BUILD

```diff
@@ -1,5 +1,11 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
+filegroup(
```

Comment:
This move looks weird. Why is it here?

Reply:
No Idea! I see this move in all files. I guess this gets auto-changed. I have deleted and regenerated vendor directory multiple times.